src/formatters/Utils.cc

```diff
@@ -159,33 +157,6 @@ bool extractOptionalFlag(std::shared_ptr<PropertySet const> const& properties, s
     return false;
 }
 
-std::string const getTableName(std::shared_ptr<Policy const> const& policy,
-                               std::shared_ptr<PropertySet const> const& properties) {
-    std::string itemName(getItemName(properties));
-    return LogicalLocation(policy->getString(itemName + ".tableNamePattern"), properties).locString();
-}
-
-std::vector<std::string> getAllSliceTableNames(std::shared_ptr<Policy const> const& policy,
-                                               std::shared_ptr<PropertySet const> const& properties) {
-    std::string itemName(getItemName(properties));
-    std::string pattern(policy->getString(itemName + ".tableNamePattern"));
-    int numSlices = 1;
-    if (properties->exists(itemName + ".numSlices")) {
-        numSlices = properties->getAsInt(itemName + ".numSlices");
-    }
-    if (numSlices <= 0) {
-        throw LSST_EXCEPT(ex::RuntimeError, itemName + " \".numSlices\" property value must be positive");
-    }
-    std::vector<std::string> names;
-    names.reserve(numSlices);
-    std::shared_ptr<PropertySet> props = properties->deepCopy();
-    for (int i = 0; i < numSlices; ++i) {
-        props->set("sliceId", i);
-        names.push_back(LogicalLocation(pattern, props).locString());
-    }
-    return names;
-}
-
```

Comment:
Some uses in **lsst/mops_nightmops**:
https://github.com/lsst/mops_nightmops/blob/master/src/MovingObjectPredictionFormatter.cc#L244
https://github.com/lsst/mops_nightmops/blob/master/src/MovingObjectPredictionFormatter.cc#L196